x-pack/plugins/lens/public/visualizations/gauge/constants.ts

```diff
@@ -33,3 +38,21 @@ export type GaugeExpressionState = GaugeStateOriginal & {
   layerId: string;
   layerType: LayerType;
 };
+
+export const gaugeTitlesByType: Record<GaugeShape, string> = {
+  [GaugeShapes.HORIZONTAL_BULLET]: i18n.translate('xpack.lens.gaugeHorizontal.gaugeLabel', {
+    defaultMessage: 'Bullet horizontal',
+  }),
+  [GaugeShapes.VERTICAL_BULLET]: i18n.translate('xpack.lens.gaugeVertical.gaugeLabel', {
+    defaultMessage: 'Bullet vertical',
+  }),
+  [GaugeShapes.ARC]: i18n.translate('xpack.lens.gaugeArc.gaugeLabel', {
+    defaultMessage: 'Gauge arc',
+  }),
+  [GaugeShapes.MAJOR_ARC]: i18n.translate('xpack.lens.gaugeTwoThirdsCircle.gaugeLabel', {
+    defaultMessage: 'Gauge major arc',
+  }),
+  [GaugeShapes.CIRCLE]: i18n.translate('xpack.lens.gaugeCircle.gaugeLabel', {
+    defaultMessage: 'Gauge circle',
+  }),
```

Comment:
I believe we should improve these terms @amyjtechwriter please take a look and give me your opinions here.
- `bullet horizontal`, `bullet vertical` should probably be rewritten as `Vertical bullet graph`, `Horizontal bullet graph`. I believe the word `graph` is required as `bullet` by itself doesn't is not used in viz
- `Gauge major arc` should represent a gauge that is a 2/3 circle. We already discussed with Nick but I'm wondering if you have a better name for that.

Reply:
Agreed but I don't like using `graph`/`chart` inconsistently. Either all viz types are referred to with `graph`/`chart` or non of them are (e.g. `Horizontal bar chart`)

Reply:
Hello hello, I agree we should keep it consistent and we currently don't say graph or chart so I think we will have to go with `bullet horizontal` and `bullet vertical`. For the gauges I think `gauge circle` is very descriptive and good. 

With the other two types we could consider:
Gauge arc --> gauge curve, gauge minor curve, gauge small curve
Gauge major arc --> gauge major curve, gauge large curve, gauge 240° curve

If the preference is `arc` over `curve` because it takes up less space I understand. I won't be offended if these alternatives are ignored for the originals @markov00 :).

Reply:
Updated naming and copy in af06fbb80efcfad88046820bec10510432e74961